tests/integration/test_vulnerability_detector/test_scan_results/data/test_cases/cases_scan_updated_package_still_vulnerable.yaml

```diff
@@ -0,0 +1,14 @@
+- name: no_alert_updated_package_still_vulnerable
+  description: When a package is updated to still vulnerable version, no alert appears showing package vuln as solved
```

Comment:
I think the following description will be more accurate:

```
Verify that when updating a vulnerable package to another version that is still vulnerable to the same CVE, the system triggers an alert for the new package installed but does not generate an alert notifying that the vulnerability was solved.
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4389/commits/18fa7b86f2c5959cfc8fd859de82e4f5ad17bd2a